ynr/context_processors.py

```diff
@@ -57,12 +57,13 @@ def add_notification_data(request):
 
     result = {}
     if request.user.is_authenticated:
-        result["photos_for_review"] = QueuedImage.objects.filter(
-            decision="undecided"
-        ).count()
-        result["suggestions_to_lock"] = SuggestedPostLock.objects.filter(
-            postextraelection__candidates_locked=False
-        ).count()
+        groups = set(request.user.groups.values_list("name", flat=True))
+        if PHOTO_REVIEWERS_GROUP_NAME in groups:
```

Comment:
This is all useful stuff being changed in this file, but the changes to this file aren't really related to the commit message. Just for the sake of future code archaeology, can we split this into 2 commits with descriptive messages.